frontend/public/style/_theme-dark.scss

```diff
@@ -0,0 +1,20 @@
+:root {
+  --os-global--palette--blue-600: var(--pf-global--palette--blue-600);
```

Comment:
If `os` is for OpenShift, I would use `co` instead (which is the prefix we use elsewhere).